xline-test-utils/README.md

```diff
@@ -0,0 +1,3 @@
+# Xline test utilities
+
+This crate provides utilities for Xline tests.
```

Comment:
We can add more words to explain why we create a new crate to put test utilities alone.

Reply:
done